### PR TITLE
chore: use non deprecated custom properties

### PR DESCRIPTION
**Pull request by montezume**
#### Summary

Just moving towards using the new custom properties 😄 😇

### CODE REVIEW

File: packages/application-components/src/components/dialogs/internals/dialog-styles.mod.css [hunk 1]
```diff
@@ -25,26 +25,32 @@
 
 .modal-content-m {
   composes: modal-content;
-  grid: [row1-start] 'top top top' minmax(var(--spacing-32), 1fr) [row1-end]
-        [row2-start] 'left main right' minmax(0, 100%) [row2-end]
-        [row3-start] 'bottom bottom bottom' minmax(var(--spacing-32), 1fr) [row3-end]
-        / minmax(var(--spacing-32), 1fr) var(--constraint-m) minmax(var(--spacing-32), 1fr);
+  grid:
+    [row1-start] 'top top top' minmax(var(--spacing-xl), 1fr) [row1-end]
```
Comment (montezume) [inline packages/application-components/src/components/dialogs/internals/dialog-styles.mod.css]: this came from running `yarn format:css`

**Comment by emmenko** [comment]
Did you update to the latest ui-kit?

**Comment by montezume** [comment]
@emmenko oh good call 🤦‍♂️

**Comment by emmenko** [comment]
The VRT test can be approved, not sure why percy thinks that it's new. It's already been approved in 2 other separate PRs...

**Pull request merged**